npf.mk

```diff
@@ -0,0 +1,55 @@
+include npfvar.mk
+
+prop : $(prop_OBJECTS)
+	ar rcv libprop.a $(prop_OBJECTS)
+
```

Comment:
Care you are in an old version of Packetgraph, think about rebase ! For example Prop is no longer in this world